test/adaptive_load/scoring_function_test.cc

```diff
@@ -0,0 +1,177 @@
+#include "envoy/common/exception.h"
+
+#include "external/envoy/source/common/config/utility.h"
+
+#include "adaptive_load/scoring_function_impl.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include <cmath>
+
+namespace Nighthawk {
+
+namespace {
+
+using nighthawk::adaptive_load::BinaryScoringFunctionConfig;
```

Comment:
Can we fully qualify these using statements? It's the recommended practice to improve code maintainability, since partially qualified statements might accidentally match a relative namespace path.

Reply:
Done.